docker/nginx/conf.d/server/server.api

```diff
@@ -21,6 +21,7 @@ client_max_body_size 128k;
 rewrite ^/portals /skynet/portals permanent;
 rewrite ^/stats /skynet/stats permanent;
 rewrite ^/skynet/blacklist /skynet/blocklist permanent;
+rewrite ^/docs(?:/(.*))?$ https://sdk.skynetlabs.com/$1 permanent;
```

Comment:
I wonder if we can avoid hard-coding the name of the portal `skynetlabs.com` here.

Reply:
I don't think we'd want that, that's our direct link for sdk that we're going to share with people and that seo should track